pkg/tso/keyspace_group_manager_test.go

```diff
@@ -713,6 +744,21 @@ func collectAssignedKeyspaceGroupIDs(re *require.Assertions, kgm *KeyspaceGroupM
 	return ids
 }
 
+func collectAllLoadedKeyspaceGroupIDs(_ *require.Assertions, kgm *KeyspaceGroupManager) []uint32 {
```

Comment:
Why not just remove the `*require.Assertions` parameter?

Reply:
removed